### PR TITLE
tesseract 5.0.1

### PR DESCRIPTION
**Pull request by jeroen**
Created with `brew bump-formula-pr`. Closes: #92440

Also included suggested changes from https://github.com/Homebrew/homebrew-core/pull/92440.

### CODE REVIEW

File: Formula/tesseract.rb [hunk 1]
```diff
@@ -21,12 +21,13 @@ class Tesseract < Formula
...
   depends_on "autoconf" => :build
-  depends_on "autoconf-archive" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "leptonica"
-  depends_on "libtiff"
+  depends_on "libarchive"
+
+  uses_from_macos "curl"
```
Comment (carlocab) [inline Formula/tesseract.rb:30]:
```suggestion
```
Let's not add this. `uses_from_macos "curl"` isn't for documentation, which is the rationale suggested in the other PR. It's to add a dependency on Homebrew `curl` on Linux. Homebrew itself requires a pre-existing `curl` to function, so we only add `uses_from_macos "curl"` whenever Homebrew's required version of `curl` is too old.

Comment (jeroen) [reply Formula/tesseract.rb]: OK I have removed it.

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Thanks @jeroen !

**Review by stweil — COMMENTED** [review]
You can also add `uses_from_macos "curl"`.

**Comment by jeroen** [comment]
@stweil we ended up not doing that. Please click the "show resolved" link above for the explanation.

**Comment by stweil** [comment]
Tesseract depends on Curl. If it is missing, the functionality is reduced. So Homebrew for Linux will miss that dependency.

**Comment by jeroen** [comment]
I think what Carlos is saying is that curl will not be missing because it part of the homebrew installer itself.

**Comment by carlocab** [comment]
If `curl` is missing `brew` won't work at all, so I would be shocked if someone manages to `brew install tesseract` with a missing `curl`

**Pull request closed without merging**